app/_how-tos/create-transit-gateway-with-operator-and-aws.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Create a Transit Gateway with {{ site.operator_product_name }} and AWS
+description: "Create a KonnectCloudGatewayTransitGateway resource with {{ site.operator_product_name }} and AWS."
+content_type: how_to
+
+
+breadcrumbs:
+  - /operator/
+  - index: operator
+    group: Konnect
+  - index: operator
+    group: Konnect
+    section: "Konnect CRDs: Cloud Gateways"
+
+products:
+  - operator
+
+works_on:
+  - konnect
+
+tags:
+  - aws
+
+prereqs:
+  inline: 
+    - title: AWS account
+      position: before
+      content: |
+          This tutorial requires an AWS account with permissions to create transit gateways and resource shares.
+      icon_url: /assets/icons/aws.svg
+    - title: AWS CLI
+      position: before
+      include_content: prereqs/aws-cli
+      icon_url: /assets/icons/aws.svg
+
+  operator:
+    konnect:
+      auth: true
+      network: true
+
+tldr:
+  q: How can I create an AWS transit gateway and link it to {{site.konnect_short_name}} using {{ site.operator_product_name }}?
+  a: Create a transit gateway in AWS and create a resources share to share the transit gateway with the AWS account linked to your {{site.konnect_short_name}} account. Then, create a [`KonnectCloudGatewayTransitGateway`](/operator/reference/custom-resources/#konnectcloudgatewaytransitgateway) and accept the transit gateway attachment in AWS.
+
+faqs:
+  - q: Can I create a {{site.konnect_short_name}} Transit Gateway linked to an Azure virtual network?
+    a: Yes, refer to [Azure peering](/dedicated-cloud-gateways/azure-peering/) to learn how to configure your VNET Peering App on Azure, then configure the [`KonnectCloudGatewayTransitGateway`](/operator/reference/custom-resources/#konnectcloudgatewaytransitgateway) resource with the [`azureTransitGateway`](/operator/reference/custom-resources/#azuretransitgateway) field.
+
```

Comment:
```suggestion
  - q: I'm getting a `unable to create transit gateway with cidr block 10.10.0.0/24 because it overlaps with cidr blocks from an existing attachment` error when I try to validate my transit gateway resource, how do I fix it?
    a: This is caused by a CIDR block overlap. Contact [Kong Support](https://support.konghq.com/support/s/) to resolve this.
```